packages/kerosene/src/fetch/transformAndCheckStatus.ts

```diff
@@ -0,0 +1,32 @@
+import transform from "./transform";
+
+declare global {
```

Comment:
Is there a way to do this without polluting global scope?

Reply:
Updated